_candidates_en/YoshitakaFujii_1.md

```diff
@@ -0,0 +1,23 @@
+---
+name:  Yoshitaka Fujii
+title: "I have developed Serverless Web application framework using DDD"
+length: 40
+audience: Intermediate
+language: Japanese
+twitter: yoshiyoshifujii
+github: yoshiyoshifujii
+icon: https://pbs.twimg.com/profile_images/907421547551277056/_vQ3f2Fg_400x400.jpg
+organization: MOTEX Inc.
+tags:
+  - Best Practices
+  - DevOps
+  - Software Design and Architecture
+  - Microservices
+suggestions:
+  - People who want to implement middle-sized Web application based on Serverless Architecture
+---
+Serverless Architecture has become widely used as an architecture saves you from variety of hard jobs such as scalability, security and Infrastructure maintenance.
+Although you can easily start Serverless Web application, maintenance cost becomes large and maintenance work gets hard as size of web application becomes large.
+To tackle this problem, I have applied Domain Driven Design to Serverless Architecture. Then I have developed a framework based on what I have learned through this process.
+In this session, I will explain how to use my framework and its design principle.
+By the way, I have took 1 year to improve my presentation in the previous ScalaMatsuri.
```

Comment:
Consider changing to "+This session will show one year's progress from my presentation at the previous ScalaMatsuri conference."